runtime/publish.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 set -ex
-for p in near-runtime-utils near-runtime-fees near-vm-errors near-vm-logic near-vm-runner near-vm-runner-standalone
+for p in near-runtime-utils near-vm-errors near-vm-logic near-vm-runner near-vm-runner-standalone
```

Comment:
Do we still use this script or cargo-workspaces completely replaced the flow?

Reply:
It looks like we're still going to use it this script for now.  `cargo-workspaces` doesn't fit well for isolated crates that you want to publish.

Reply:
@matklad @willemneal @mikedotexe can provide more details. They had a call today on this.